diff --git a/src/collectives/device/Makefile b/src/collectives/device/Makefile
index 3796fb1..a5485b8 100644
--- a/src/collectives/device/Makefile
+++ b/src/collectives/device/Makefile
@@ -50,6 +50,7 @@ $(STATICLIB): $(LIBOBJ) $(DEVOBJ)
 # be regenerated.
 $(OBJDIR)/%.dep : %.cu
 	@mkdir -p $(OBJDIR)
+	@echo $(NVCC) $(NVCUFLAGS) 
 	@$(NVCC) $(NVCUFLAGS) -M $< -o $@.tmp
 	@sed "0,/^.*:/s//$(subst /,\/,$@):/" $@.tmp > $@
 	@sed -e 's/.*://' -e 's/\\$$//' < $@.tmp | fmt -1 | \
diff --git a/src/collectives/device/gen_rules.sh b/src/collectives/device/gen_rules.sh
index 97dc0ae..9b74d8c 100755
--- a/src/collectives/device/gen_rules.sh
+++ b/src/collectives/device/gen_rules.sh
@@ -1,4 +1,4 @@
-#!/bin/bash
+#! /usr/bin/env bash
 #
 # Copyright (c) 2018-2020, NVIDIA CORPORATION. All rights reserved.
 #
